DockerfileNameRegistry

```diff
@@ -0,0 +1,15 @@
+# Build Agora from source
+FROM bpfk/agora-builder:latest AS RegistryBuilder
+ARG DUB_OPTIONS
```

Comment:
I don't think you need this (or `${DUB_OPTIONS}`). It was put to allow us to build Agora with coverage on in the integration tests.